ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -450,19 +494,21 @@ func (pg *DEXOnboarding) subStepAddServer(gtx C) D {
 		layout.Rigid(func(gtx C) D {
 			return layout.Inset{Top: u16}.Layout(gtx, pg.serverCertEditor.Layout)
 		}),
-		layout.Rigid(func(gtx C) D {
-			return pg.formFooterButtons(gtx)
-		}),
+		layout.Rigid(pg.formFooterButtons),
 	)
 }
 
 // formFooterButtons is a convenience function that prepares the required
 // buttons for each form page.
 func (pg *DEXOnboarding) formFooterButtons(gtx C) D {
-	var addBackBtn bool
+	addBackBtn, nextBtnEnabled, backBtnEnabled := true, true, true
 	switch pg.currentStep {
-	case onboardingPostBond, onboardingChooseServer, onBoardingStepAddServer:
-		addBackBtn = true
+	case onboardingSetPassword:
+		addBackBtn = false
+	case onboardingChooseServer, onBoardingStepAddServer:
+		backBtnEnabled = !pg.dexc.IsInitialized()
```

Comment:
I can't go back if I have initialized and I'm on the choose/add server step? I'm not really getting this part.

Reply:
`core.DiscoverAccount` requires that dex has been initialised (i.e. pass has been set).